src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -392,7 +392,7 @@ public override bool ReflectionOnly
         // given name.  (Name should not include path.)
         public override FileStream? GetFile(string name)
         {
-            RuntimeModule? m = (RuntimeModule?)GetModule(name);
+            RuntimeModule? m = GetModule(name) as RuntimeModule;
```

Comment:
Why change this?  Was it not compiling for some reason?

Reply:
its not showing error, i think it wasn't like this when i annotating, will revert